ansible/vars/topo_t0-116.yml

```diff
@@ -0,0 +1,189 @@
+topology:
+  host_interfaces:
+   - 0
+   - 1
+   - 2
+   - 3
+   - 4
+   - 5
+   - 6
+   - 7
+   - 8
+   - 9
+   - 10
+   - 11
+   - 12
+   - 13
+   - 14
+   - 15
+   - 16
+   - 18
+   - 20
+   - 21
+   - 22
+   - 23
+   - 24
+   - 25
+   - 26
+   - 27
+   - 28
+   - 29
+   - 30
+   - 31
+   - 32
+   - 33
+   - 34
+   - 35
+   - 36
+   - 37
+   - 38
+   - 39
+   - 40
+   - 41
+   - 42
+   - 43
+   - 52
+   - 53
+   - 54
+   - 55
+   - 56
+   - 57
+   - 58
+   - 59
+   - 60
+   - 61
+   - 62
+   - 63
```

Comment:
why only 64 host interfaces?

Reply:
It is a temporary measure to let the IO test run with current deployment scripts. Once the deployment scripts get updated to allow second 50G ports, this topology will change to include all interfaces.